### DISCUSSION TITLE
new idea, in addition to time/size, compare also last block

### DISCUSSION THREAD

**Discussion by anttiryt** [RsyncProject/rsync#422] [Ideas]
Through my years as sysop, I've come to not entirely rely on filename time and size. Sometimes the contents are wrong. There's already checksum comparison option, but that is quite time consuming on large devices. My proposition is to add an option to do checksum only on the last block. This is quite reliable to see if copy was hang midprocess.

**Comment by ericmarceau-rogers** [comment]
I very much like that concept of your proposal, but maybe have an additional option for what fraction of the file to use, like maybe last 5%, last 20%.
